csrc/kernel_cache.cpp

```diff
@@ -1208,37 +1199,6 @@ std::vector<at::Tensor> FusionKernelRuntime::runKernelWithInput(
   }
   auto outputs = executor.runFusion(args, launch_params, compile_params);
 
-  // Accumulate the kernel time of each segment
-  kernel_time_ms_ += executor.kernelTimeMs();
-
-  // Print relevant information all at once for easy debuging of perf
-  if (isDebugDumpEnabled(DebugDumpOption::PerfDebugVerbose) &&
```

Comment:
Is the option `DebugDumpOption::PerfDebugVerbose` redundant as well?
Then we should remove it as well.

User although may still like to print out this information at once.

Reply:
`PerfDebugVerbose` allows for debugging a range of things like segmentation and occupancy beyond what is captured in the profiler.  I was only concerned with cleaning up the redundancy I had made in order to keep kernel time and effective bandwidth measurements working while I was developing.